transform/app/transform/utils/delete.py

```diff
@@ -0,0 +1,26 @@
+import os
+import logging
+import shutil
+
+from app.settings import settings
+
+logger = logging.getLogger(__name__)
+
+
+def clean_tmp():
+    """
+    Deletes all files in the _tmp directory.
+    """
+    for filename in os.listdir(
```

Comment:
Your approach is indeed valid. As a curiosity - I wrote something really similar using os.walk.  Alternative approach:
```
def remove_files(path):
    for root, dirs, files in os.walk(path):
        for file in files:
            file_path = os.path.join(root, file)
            os.remove(file_path)
        for dir in dirs:
            dir_path = os.path.join(root, dir)
            os.rmdir(dir_path)
```

Using os.walk():
- Recursively walks through the directory tree, which is useful when you want to perform operations on files and directories at different levels of the hierarchy. And handle them somewhat differently. Your approach lists the content. 

It is just a side note. You dont need to do anything

Reply:
I was thinking about a function that was only designed to clean TMP. After finding the solution I didn't try to find better and cleaner. We could use your code to have recursive function.